packages/manager/apps/web/client/app/core/translations/Messages_fr_FR.json

```diff
@@ -122,7 +122,8 @@
   "common_ipv4": "IPv4",
   "common_ipv6": "IPv6",
   "common_bitFormat": "bits",
-  "common_service_expired": "Le service {{t0}} n'a pas été renouvelé. Cela est peut être dû à une décision de votre part ou un retard de paiement de votre côté.",
+  "common_service_expired_1": "Ce service {{t0}} est suspendu. Rendez-vous dans la section",
+  "common_service_expired_2": "pour le réactiver.",
```

Comment:
For the translation split, more information:

The whole content `Rendez-vous dans la section Mes services pour le réactiver` should not be splitted here. The goal is to have two different sentences as following:

- `Ce service {{t0}} est suspendu`. First one.
- `Rendez-vous dans la section Mes services pour le réactiver`. Second one, 

Keeping a full sentence is necessary to avoid losing sense when translating the content in other languages as we don't know, for example if `pour le réactiver` can be correctly translated into German.

Then, the `<a>` link should contains the whole second translation, to keep HTML out of the JSON files

Reply:
@JeremyDec , ill check with reporter if this is okay.

Reply:
Has been changed